pkg/controller/authentication/containers.go

```diff
@@ -984,17 +984,6 @@ func buildIdentityManagerContainer(instance *operatorv1alpha1.Authentication, id
 			Name:  "MASTER_HOST",
 			Value: icpConsoleURL,
 		},
-		{
-			Name: "NAMESPACES",
-			ValueFrom: &corev1.EnvVarSource{
```

Comment:
there are two references to NAMESPACES in identity-manager but they are not used in the CP3 context, I will test with the merge build and confirm that nothing breaks